agent/agent.go

```diff
@@ -249,17 +248,10 @@ func (a *Agent) finishTask(ctx context.Context, tc *taskContext, status string,
 	return resp, nil
 }
 
-func (a *Agent) getCurrentCommandType(tc *taskContext) string {
-	if tc.taskConfig != nil {
-		return tc.currentCommand.Type()
-	}
-	return model.DefaultCommandType
-}
```

Comment:
just to confirm, the "set to default if not set" is happening in command.Render() now?

Reply:
I'm not sure where this is happening. I removed this function because nothing was calling it.

Reply:
I think that we might not be setting the command type if we don't have this logic somewhere, but we could roll it into endTaskResponse.

Reply:
Done.